### PR TITLE
[chart] Add PSP for logger

### PR DESCRIPTION
**Pull request by LittleFox94**
This pull requests adds a flag to generate a PodSecurityPolicy for the logger.

I have PodSecurityPolicies enabled in my cluster and run the logger pods in privileged mode. They were not able to start before this patch.

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1568)

### CODE REVIEW

File: charts/fission-all/templates/fluentbit.yaml [hunk 1]
```diff
@@ -19,6 +19,64 @@ data:
...
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    svc: logger
+roleRef:
+  kind:     Role
+  name:     psp:{{ .Release.Name }}-fission-logger-privileged
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind:      ServiceAccount
+  name:      default
```
Comment (life1347) [inline charts/fission-all/templates/fluentbit.yaml:77]: The service account logger used is `fission-svc`, do we need to change it from `default` to `fission-svc`?

**Comment by LittleFox94** [comment]
The permission to create privileged pods is required by the daemonset-controller, which uses the default sa

(cannot reply to the comment with my app)

**Comment by life1347** [comment]
ok, thanks for the clarification. I will find some time to test the changes.

**Comment by life1347** [comment]
@vishal-biyani I'm quite busy recently, is there any chance for you to test this PR? Thanks

**Comment by vishal-biyani** [comment]
@life1347 Yes I will review and test it.

**Comment by vishal-biyani** [comment]
Hey, @LittleFox94 Tested and found good for merging. The only question I have is `privileged` absolutely necessary given the other four rules are `RunAsAny`? ( I am not yet super-familiar with PSP)

**Comment by LittleFox94** [comment]
Running those pods in privileged mode is another flag in the helm chart. I seem to need that on my cluster (though it was late at night, will check again), but there are definitely people who will need it (I think Openshift is strange here)

This psp is actually only a fix for the existing flag on clusters with psp enabled

**Comment by vishal-biyani** [comment]
Ah, nice. I did not notice that earlier - but thanks a lot for clarifying! Keep safe in these times and I will merge this now.

**Comment by LittleFox94** [comment]
Thanks, and keep safe yourself :)

**Pull request merged**